win/cross-build-minimal-ffmpeg

```diff
@@ -35,7 +35,13 @@ configure() {
               --disable-encoders \
               --enable-encoder=pcm_s16le \
               --disable-decoders \
-              $(ffmpeg -hide_banner -codecs | grep '^ D.A' | cut -d ' ' -f 3 | xargs -I'{}' echo --enable-decoder={}) \
```

Comment:
On my system, the previous command enables 189 codecs, you are leaving only 7 here, so this is clearly not equivalent.
For `build-minimal-ffmpeg`, this does not matter because it is only used for tests, but `cross-build-minimal-ffmpeg` is used to generate the Windows release archives, so I'd rather have maximum codec coverage.

Reply:
Ok, I changed it to derive the list of all audio decoders from `libavcodec/allcodecs.c`. It now actually enables a couple more decoders than before, but that shouldn't be an issue.